### ISSUE TITLE
yard server doesn't show any classes or methods for gems

### ISSUE THREAD

**Issue by coldnebo** [lsegal/yard#1018]
I'm trying to write doc for a gem, but `yard server` in the gem directory doesn't show classes or methods. It only shows "Top Level Namespace" in the nav, and the README.md in the right pane. I can get to Files and see other files like USAGE.md and LICENSE.txt, but nothing else.

Yet, when I run `yard doc`, from the gem directory, everything works and `./doc/index.html` has all the classes and methods as expected.

I've tried looking for yardopts to include other files, however adding `lib/**/*.rb` to .yardopts merely includes the source verbatim instead of producing doc for the source.

What am I doing wrong?

## Environment details:

* OS: Ubuntu
* Ruby version (`ruby -v`): 2.1.7
* YARD version (`yard -v`): 0.9.5

**Comment by coldnebo** [comment]
And now, it's magically back without any changes.

Is there an assumption that the server is run from a local filesystem?  I'm running it from a network filesystem in this case, and I'm wondering if that could be a problem.

Anyway, I guess it works for now. Wasted a lot of time on this.  >:(

**Comment by lsegal** [comment]
Can you show steps to reproduce this?

**Comment by coldnebo** [comment]
I described the only steps I know.  The problem may have gone away when I cleared the .yardoc directory, so it's possible that some kind of caching issue prevented the file doc from being read.

Just from superficially skimming the yard server code, I can't see where files are actually parsed and served... my guess is that there is a difference between `yard server` and `yard doc` but I'd need a clue before being able to guess what's going on to reproduce.  Alas, I don't have time now, so I'll go ahead and close this for now.
